myosuite/envs/myo/assets/leg/myoosl_runtrack.xml

```diff
@@ -38,10 +38,10 @@
 
     <keyframe>
         <key name='stand' qpos='0 14 .95 0.707388 0 0 -0.706825 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0'/>
+        <key name='osl_backward' qpos='0 14 0.86  0.7011 0.0923 0.0923 -0.7011     -0.233 -0.0279385 -0.041886 0 0 0 0 0.377 0.1672 0.785 -0.0279385 -0.041886 0.00247908 0.0010198 0.209 0.0275069 0.136817 -0.0524 -0.00117055 -0.000125295 -0.0302192 0.0395202 -0.194029' qvel='0 -1.2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0'/>
```

Comment:
for merging with runtrack P2, also add this to the runtrack_p2 xml that you'll find there